### PR TITLE
添加头像

### PR DESCRIPTION
**Pull request by Akarinnnnn**
最大大小为3x3cm

### CODE REVIEW

**Comment by Akarinnnnn** [comment]
#2

File: src/components/resume/Header.vue [hunk 1]
```diff
@@ -13,9 +13,12 @@ const name = computed(() => {
...
 <template>
   <div v-if="resume">
-    <h1 class="font-normal text-center text-4xl">
-      {{ name || 'Your Name' }}
-    </h1>
+    <div class="flex justify-between">
+      <span class="font-normal text-center text-4xl">
+        {{ name || 'Your Name' }}
+      </span>
+      <img v-if="resume.basics.avatar" :src="resume.basics.avatar" style="max-height: 3cm; max-width: 3cm;">
```
Comment (YunYouJun) [inline src/components/resume/Header.vue]: I think we can use atomic css directly.

`max-h-3cm`

**Review by YunYouJun — CHANGES_REQUESTED** [review]

File: src/components/resume/Header.vue (hunk 1 above)
Comment (Akarinnnnn) [reply src/components/resume/Header.vue]: 已修复，另外git hook缺少ts-node-script导致无法Commit

Comment (YunYouJun) [reply src/components/resume/Header.vue]: `max-width` is also needed.

**Comment by YunYouJun** [comment]
image

When i comment `avatar`, the name is not centered properly, which breaks the style without an avatar.

**Pull request closed without merging**